lib/chef/provisioning/vsphere_driver/vm_helper.rb

```diff
@@ -17,12 +17,6 @@ class VmHelper
       Timeout::Error, IPAddr::AddressFamilyError
     ].freeze
 
-    # If the IP is true
```

Comment:
Removed code as there was only 1 reference to it, in the `ip_to_bootstrap` method